.chloggen/transform-name-suffix-semantics.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: processor/transform
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Change default metric name suffix for convert summary and extract histogram functions according to OTel semantic conventions"
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [33850]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: "Adaptations of the metric name suffix were made to match the latest OTel convention which uses '.' instead of '_' in suffix"
```

Comment:
```suggestion
subtext: "Suffixes are now separated with '.' instead of '_'. Specifically, many created metric names now end in `.count` or `.sum`."
```